packages/cli/tests/lib/cli.js

```diff
@@ -1,13 +1,16 @@
 const { join } = require('path');
-const { mkdirSync, symlinkSync } = require('fs');
+const { mkdir, symlink } = require('fs').promises;
```

Comment:
Should be ever so slightly faster with promises, and everything is already async anyways.